apps/prairielearn/src/pages/administratorQuery/administratorQuery.html.ts

```diff
@@ -266,7 +267,7 @@ export function AdministratorQuery({
                       <thead>
                         <tr>
                           ${query_run.result.columns?.map((col) => {
-                            return html` ${renderHeader(query_run.result.columns, col)} `;
+                            return html` ${renderHeader(query_run.result?.columns, col)} `;
```

Comment:
Nit: this could return the result of `renderHeader` directly; ditto for `render(...)` below.

Reply:
Done in 358447d03. Also took the opportunity to simplify a few expressions.